docs/guides/create_service_existing_agent.md

```diff
@@ -102,7 +107,7 @@ You can view the agent contents stored in the IPFS [here](https://gateway.autono
               chain_id: 31337
               poa_chain: false
               default_gas_price_strategy: eip1559
-
+              
```

Comment:
The hello world service.yaml file has changed in a PR yesterday (I forgoto to update this file accordingly). Please update this content to match the file.

Reply:
was this addressed?